go/src/koding/go-webserver/envdata.go

```diff
@@ -17,10 +17,10 @@ import (
 const myPermissionsAndRolesPath = "/-/my/permissionsAndRoles"
 
 var defClient = httputil.NewClient(&httputil.ClientConfig{
-	DialTimeout:           1 * time.Second,
-	RoundTripTimeout:      2 * time.Second,
-	TLSHandshakeTimeout:   2 * time.Second,
-	ResponseHeaderTimeout: 2 * time.Second,
+	DialTimeout:           30 * time.Second,
```

Comment:
I think DialTimeout is not affected by ELB timeout, since it's time needed to connect to ELB. But definitely RoundTripTimeout can be lowered to 20s since the conn is going to be dropped after 20s either way by ELB.